src/profile-logic/import/dhat.js

```diff
@@ -199,26 +226,31 @@ export function attemptToConvertDhat(json: mixed): Profile | null {
     let column = null;
 
     const result = funcName.match(
-      /^0x([0-9a-f]+): (.+) \((.+):(\d+):(\d+)\)$/i
+      /^0x([0-9a-f]+): (.+) \((.+?)(?::(\d+))?(?::(\d+))?\)$/i
     );
-    // ^0x([0-9a-f]+): (.+) \((.+):(\d+):(\d+)\)$   Regex
-    //    (1        )  (2 )   (3 ) (4  ) (5  )      Capture groups
-    // ^                                        $   Start to end
-    //               :      \(                \)    Some raw characters
-    //    ([0-9a-f]+)                               Match the address, e.g. 10250148c
-    //                 (.+)                         Match the function name
-    //                        (.+)                  Match the filename
-    //                             (\d+)            Match the line number
-    //                                   (\d+)      Match the column number
+    // ^0x([0-9a-f]+): (.+) \((.+?)(?::(\d+))?(?::(\d+))?\)$ Regex
+    //    (1        )  (2 )   (3  )    (4  )      (5  )        Capture groups
+    // ^                                                   $   Start to end
+    //               :      \(                           \)    Some raw characters
+    //    ([0-9a-f]+)                                          Match the address, e.g. 10250148c
+    //                 (.+)                                    Match the function name
+    //                        (.+?)                            Match the filename
+    //                             (?:      )?                 Optionally include the line
+    //                                 (\d+)                   Match the line number
+    //                                        (?:      )?      Optionally include the column
+    //                                            (\d+)        Match the column number
 
     // Example input: "0x10250148c: alloc::vec::Vec<T,A>::append_elements (vec.rs:1469:9)"
     // Capture groups:   111111111  2222222222222222222222222222222222222  333333 4444 5
+
+    // Example input: "0x484DE30: memalign (in /usr/libexec/valgrind/vgpreload_dhat-amd64-linux.so)"
```

Comment:
Should also have the regexp remove the word `in`? I see it's present in the output:
![image](https://github.com/user-attachments/assets/47a1f500-e249-4098-8a56-54d3f574599c)

For example you can add `(?:in )?` at the start of the parenthesis. Or explicitely remove it later.

nit: I see in your example file that the line number can be present without the column number, can you add this example?
eg `"0x58E297: marian::io::binary::loadItems(void const*, std::vector<marian::io::Item, std::allocator<marian::io::Item> >&, bool) (binary.cpp:74)"`